Source/DafnyStandardLibraries/Makefile

```diff
@@ -11,8 +11,8 @@ DOO_FILE_ARITHMETIC_TARGET=binaries/DafnyStandardLibraries-arithmetic.doo
 all: check-binary test check-format check-examples
 
 verify:
-	$(DAFNY) verify src/dfyconfig.toml
-	$(DAFNY) verify src/dfyconfig-arithmetic.toml
+	$(DAFNY) verify src/DafnyStdLibs/EnableNonLinearArithmetic/dfyconfig.toml
```

Comment:
Good fix. Should probably delegate to a similar `verify` target on the `TargetSpecific` Makefile too for consistency with the other targets.